lib/gds_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module GdsApi
-  VERSION = '49.3.1'.freeze
+  VERSION = '49.3.2'.freeze
```

Comment:
My interpretation on semver is that this is a minor change, and not a patch as it doesn't fix any bugs [1], but I'm happy either way, as the versioning on this repository isn't consistent or very important anyway.

1: `PATCH version when you make backwards-compatible bug fixes.`

Reply:
@cbaines Fair enough. I'll make it a minor version change. Thanks.